lib/oli_web/live/delivery/student_onboarding/wizard.ex

```diff
@@ -107,6 +107,7 @@ defmodule OliWeb.Delivery.StudentOnboarding.Wizard do
           module={Stepper}
           steps={@steps}
           current_step={@current_step_index}
+          on_cancel={JS.navigate(Routes.delivery_path(OliWeb.Endpoint, :open_and_free_index))}
```

Comment:
I think this makes sense for open and free/independent sections but what about LTI sections? Maybe we shouldn't show this button in that case?

Reply:
You're right. Good catch!